### ISSUE TITLE
hook to open desired set of windows at startup

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#522]
Customer request. If out of scope for November then remove lable ... not a priority

**Comment by ericwinger** [comment]
@dalehenrich Is the request to open windows at image startup (i.e. clicking on the .exe file) or post-login?   Can I assume post-login?

**Comment by ericwinger** [comment]
And do we know which windows?  Project browser? Inspector? Method list browser? other?

**Comment by ericwinger** [comment]
Should have a fix for this in 3.0.78. Syntax looks like this currently: 

```
transcriptEnabled: true
#openBrowserOnClass: RowanAnsweringService
openBrowserOnPackage: Rowan-Services-Core
#openBrowserOnClass: Rowan 
#openSUnitBrowserOnPackage: Rowan-Services-Tests
```
# comments out. 
Only methods understood by ``JadeitePreferences`` are run so adhoc commands cannot be setup. 

Also, exposed the preferences file on the console's browse menu

**Comment by LisaAlmarode** [comment]
Looks fine in 3.0.79
